lib/client.js

```diff
@@ -246,11 +244,7 @@ class AssetComputeClient extends EventEmitter {
      */
     async process(source, renditions, userData) {
         if (!this._registered) {
-            await this.register();
-            const waitTime = process.env.REGISTER_WAIT_TIME_MSEC || DEFAULT_REGISTER_WAIT_TIME_MSEC;
-            console.log(`Registered a new journal. Waiting ${waitTime}s before calling /process`);
-            const sleep = require('util').promisify(setTimeout);
-            await sleep(waitTime);
+            throw new Error('Must call register before calling /process');
```

Comment:
👍  Makes it clear if developers are not using the API as intended.

Maybe add the note about using `createAssetComputeClient()` as a simpler alternative.

Reply:
Done.

Reply:
I determined it is fine to break compatibility since the only to repos that use this are IT tests (which I already changed in my branch to use the new version) and the devtool (which I'll fix)